worlds/cvcotm/presets.py

```diff
@@ -0,0 +1,159 @@
+    "require_all_bosses":            RequireAllBosses.option_false,
+    "early_double":                  EarlyDouble.option_false,
+    "death_link":                    DeathLink.option_off,
+    "completion_goal":               CompletionGoal.option_dracula,
+}
+
+competitive_all_bosses_options = {
+    "progression_balancing":         ProgressionBalancing.default,
+    "accessibility":                 Accessibility.option_items,
+    "ignore_cleansing":              IgnoreCleansing.option_false,
+    "auto_run":                      AutoRun.option_false,
+    "dss_patch":                     DSSPatch.option_true,
+    "always_allow_speed_dash":       AlwaysAllowSpeedDash.option_true,
+    "break_iron_maidens":            BreakIronMaidens.option_false,
+    "required_last_keys":            8,
+    "available_last_keys":           8,
+    "buff_ranged_familiars":         BuffRangedFamiliars.option_true,
+    "buff_sub_weapons":              BuffSubWeapons.option_true,
+    "buff_shooter_strength":         BuffShooterStrength.option_false,
+    "item_drop_randomization":       ItemDropRandomization.option_hard,
+    "halve_dss_cards_placed":        HalveDSSCardsPlaced.option_true,
+    "countdown":                     Countdown.option_none,
+    "sub_weapon_shuffle":            SubWeaponShuffle.option_true,
+    "disable_battle_arena_mp_drain": DisableBattleArenaMPDrain.option_false,
+    "require_all_bosses":            RequireAllBosses.option_true,
+    "early_double":                  EarlyDouble.option_false,
+    "death_link":                    DeathLink.option_off,
+    "completion_goal":               CompletionGoal.option_dracula,
+}
+
+hardcore_mode_options = {
+    "progression_balancing":         0,
+    "accessibility":                 Accessibility.option_locations,
+    "ignore_cleansing":              IgnoreCleansing.option_true,
+    "auto_run":                      AutoRun.option_false,
+    "dss_patch":                     DSSPatch.option_true,
+    "always_allow_speed_dash":       AlwaysAllowSpeedDash.option_false,
+    "break_iron_maidens":            BreakIronMaidens.option_false,
+    "required_last_keys":            9,
+    "available_last_keys":           9,
+    "buff_ranged_familiars":         BuffRangedFamiliars.option_false,
+    "buff_sub_weapons":              BuffSubWeapons.option_false,
+    "buff_shooter_strength":         BuffShooterStrength.option_false,
+    "item_drop_randomization":       ItemDropRandomization.option_hard,
+    "halve_dss_cards_placed":        HalveDSSCardsPlaced.option_true,
+    "countdown":                     Countdown.option_none,
+    "sub_weapon_shuffle":            SubWeaponShuffle.option_true,
+    "disable_battle_arena_mp_drain": DisableBattleArenaMPDrain.option_false,
+    "require_all_bosses":            RequireAllBosses.option_false,
+    "early_double":                  EarlyDouble.option_false,
+    "death_link":                    DeathLink.option_on,
```

Comment:
Highly recommend not including death_link in the hardcore_mode_options. Death link isn't really a difficulty option, more of a silly thing.
Mainly, how many reports of "I died randomly" or "I got death linked even though I didn't turn death link on" do you think you will receive before this will need to be changed?